scss/_type.scss

```diff
@@ -148,3 +148,30 @@ mark,
     content: "\00A0 \2014"; // nbsp, em dash
   }
 }
+
+@include media-breakpoint-down(xs) {
+  // decrease font size on portrait phones and smallest devices
+
+  .lead {
+    font-size: $lead-font-size/1.1;
```

Comment:
`$lead-font-size/1.1` should be written with a single space on each side of the operator: `$lead-font-size / 1.1`